recipes/intreehooks/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0" %}
+
+package:
+  name: intreehooks
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/i/intreehooks/intreehooks-{{ version }}.tar.gz
+  sha256: 87e600d3b16b97ed219c078681260639e77ef5a17c0e0dbdd5a302f99b4e34e1
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - wheel
+    - pip
+  run:
+    - python
+    - pytoml
+
+test:
+  imports:
+    - intreehooks
+
+about:
+  home: https://github.com/takluyver/intreehooks
+  license: MIT
+  license_family: MIT
+  # https://github.com/takluyver/intreehooks/issues/1
+  # license_file: LICENSE
```

Comment:
We need to ship the license file even if an issue is made IIUIC. Let's make sure to add it here.

Reply:
A generic MIT license without the copyright header may be as problematic as a missing license, no?

Reply:
Ack. Sorry. I misunderstood the situation. Ignore this. It is clear this is supposed to be MIT.

Reply:
No problem. I'll ping the author upstream to solve this ASAP.

Reply:
Yup. LGTM and merged!

Reply:
In that case, we are not allowed to package this as a conda package.

Reply:
We have packaged code like this before when it was clear what was meant in the upstream repo. If you think we should remove the package, then we can delete from master. Staged recipes conversion is broken anyways.

Reply:
> We have packaged code like this before when it was clear what was meant in the upstream repo.

That's just wrong. A condition of MIT license is that the copyright notice is distributed, but if there's no copyright, we just can't distribute.

Reply:
OK. I had consulted with core before when doing this, but happy to not as well. Thanks!